cmd/queue/main.go

```diff
@@ -64,6 +63,10 @@ const (
 	// in the mesh.
 	quitSleepDuration = 20 * time.Second
 
+	// Set equal to the queue-proxy's ExecProbe timeout to take
+	// advantage of the full window
+	probeTimeout = 10 * time.Second
```

Comment:
Should we make this a flag so it can be kept consistent with TimeoutSeconds in the probe configuration?  Maybe we shouldn't do this until we pass the entire probe descriptor through?

Reply:
Yeah, the idea was that this was separate from the issue of passing in the full probe to have something user-defined applied via the queue-proxy. There are a bunch of things in this file that will change once we start passing all of the configuration in.